src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -264,6 +306,10 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
     }
   }, [fetchFarmingOpportunities, fetchLpOpportunity, transaction])
 
+  const foxFarmingTotalCryptoAmount = foxFarmingOpportunities.reduce((acc, opportunity) => {
+    return acc.plus(bnOrZero(opportunity.cryptoAmount))
+  }, bnOrZero(0))
```

Comment:
Nitpick again, but would rather get a string output from that `reduce()` if not memoized (every `Array.prototype.reduce()` call across renders will create a new reference - since react uses triple-equal (`===`) comparison, anything that's not a primitive will cause re-renders